### DISCUSSION TITLE
drop collection - does this not qualify as HTTP "delete" verb?

### DISCUSSION THREAD

**Discussion by PaulAnderson-One** [milvus-io/milvus#33668] [Ideas & Feature requests]
I searched for "drop" in the discussions - pardon me if there is a wider scope discussion about REST URIs
the API doc shows

POST `https://${MILVUS_URI}/v1/vector/collections/drop` for dropping a collection (body not shown)

I'm wondering what the reason is that this wasn't modeled as

DELETE `https://${MILVUS_URI}/v1/vector/collections/{collectionName}`
this seems more concise and the verb leaves no doubt about what the caller is about to do

**Comment by yhmo** [comment]
I think it is implementor's design. 
In V2 RESTful, all the requests are defined as POST.
To make the urls similar to sdk interfaces, we include verbs in urls, although we know it is not a good design for RESTful.
